addons/stock/migrations/12.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,196 @@
+NEW ir.model.access: stock.access_stock_rule_stock_manager
+NEW ir.model.access: stock.access_stock_rule_user
+NEW ir.model.access: stock.access_uom_category_stock_manager
+NEW ir.model.access: stock.access_uom_uom_stock_manager
+DEL ir.model.access: stock.access_procurement_rule
+DEL ir.model.access: stock.access_procurement_rule_internal
+DEL ir.model.access: stock.access_procurement_rule_manager
+DEL ir.model.access: stock.access_procurement_rule_stock_manager
+DEL ir.model.access: stock.access_procurement_rule_user
+DEL ir.model.access: stock.access_product_attribute_line_manager
+DEL ir.model.access: stock.access_product_attribute_price_manager
+DEL ir.model.access: stock.access_product_uom_categ_stock_manager
+DEL ir.model.access: stock.access_product_uom_stock_manager
+DEL ir.model.access: stock.access_stock_incoterms_all
+DEL ir.model.access: stock.access_stock_incoterms_manager
+DEL ir.model.access: stock.access_stock_location_path_internal_user
+DEL ir.model.access: stock.access_stock_location_path_stock_user
+DEL ir.model.access: stock.access_stock_location_path_user
+DEL ir.rule: stock.stock_location_path_comp_rule (noupdate)
+NEW ir.ui.menu: stock.menu_action_rules_form
+DEL ir.ui.menu: stock.menu_action_incoterm_open
+NEW ir.ui.view: stock.exception_on_picking
+NEW ir.ui.view: stock.package_level_form_view
+NEW ir.ui.view: stock.package_level_tree_view_picking
+NEW ir.ui.view: stock.report_lot_label
+NEW ir.ui.view: stock.report_picking_type_label
+NEW ir.ui.view: stock.report_stock_rule
+NEW ir.ui.view: stock.report_stock_rule_left_arrow
+NEW ir.ui.view: stock.report_stock_rule_line
+NEW ir.ui.view: stock.report_stock_rule_right_arrow
+NEW ir.ui.view: stock.report_stock_rule_suspension_points
+NEW ir.ui.view: stock.stock_fixed_putaway_strat_form
+NEW ir.ui.view: stock.stock_package_destination_form_view
+NEW ir.ui.view: stock.stock_product_search_form_view
+NEW ir.ui.view: stock.stock_scrap_view_kanban
+NEW ir.ui.view: stock.view_product_replenish
+NEW ir.ui.view: stock.view_route_rule_form
+NEW ir.ui.view: stock.view_stock_rule_filter
+NEW ir.ui.view: stock.view_stock_rule_form
+NEW ir.ui.view: stock.view_stock_rule_tree
+NEW ir.ui.view: stock.view_stock_rules_report
+NEW ir.ui.view: stock.view_stock_track_confirmation
+DEL ir.ui.view: stock.inventory_planner
+DEL ir.ui.view: stock.report_lot_barcode
+DEL ir.ui.view: stock.stock_incoterms_form
+DEL ir.ui.view: stock.stock_incoterms_view_search
+DEL ir.ui.view: stock.stock_location_path_filter
+DEL ir.ui.view: stock.stock_location_path_form
+DEL ir.ui.view: stock.stock_location_path_tree
+DEL ir.ui.view: stock.view_incoterms_tree
+DEL ir.ui.view: stock.view_procurement_rule_filter
+DEL ir.ui.view: stock.view_procurement_rule_form
+DEL ir.ui.view: stock.view_procurement_rule_tree
+DEL ir.ui.view: stock.view_quant_package_picking_tree
+NEW res.groups: stock.group_lot_on_delivery_slip
+DEL res.request.link: stock.req_link_tracking (noupdate)
+DEL stock.incoterms: stock.incoterm_CFR
```

Comment:
Is this renamed somewhere?

Reply:
Don't know. I just simply did "stock.incoterms -> account.incoterms" rename in `account` migration. I didn't check all records of stock.incoterms. Maybe some of them went to localization modules.

Reply:
Please check. They are standard, so they should be somewhere else apart from localization modules. The case is that I remember some `ADD` of these records in one review, but I didn't take that into account...

Reply:
Checked, all of them are in `account`. Happens that in the last openupgrade analysis update the rename of this model was not yet in apriori.